src/pages/signin/ResendValidationForm.js

```diff
@@ -21,6 +21,9 @@ const propTypes = {
 
         // Weather or not the account is validated
         validated: PropTypes.bool,
+
+        // Weather or not the account is closed
```

Comment:
This `Weather` (and the above) should be `Whether` ([resource](https://www.grammar-monster.com/easily_confused/weather_whether_wether.htm#:~:text=Weather%20refers%20to%20the%20state,wonder%20whether%20it%20will%20rain.))

Reply:
lol copy paste from the previous one.
.... which was also done by me 🙈